src/viam/sdk/common/utils.hpp

```diff
@@ -47,5 +47,15 @@ std::string bytes_to_string(std::vector<unsigned char> const& b);
 std::chrono::microseconds from_proto(const google::protobuf::Duration& proto);
 google::protobuf::Duration to_proto(const std::chrono::microseconds& duration);
 
+/// @brief Set the boost trivial logger's severity depending on args.
+/// @param argc The number of args.
+/// @param argv The commandline arguments to parse.
+///
+/// Sets the boost trivial logger's severity to debug if "--log-level=debug" is the
+/// the third argument. Sets severity to info otherwise. Useful for module
```

Comment:
(opt) if we end up updating this func, we should update comment here to remove the "third arg" restriction.